dev_guide/volumes.adoc

```diff
@@ -306,3 +314,51 @@ List volume *v1* defined on all deployment configurations:
 ----
 $ oc volume dc --all --name=v1
 ----
+
+[[volumes-specifying-a-subpath]]
+== Specifying a Sub-path
+
+Use the `volumeMounts.subPath` property to specify a `subPath` inside a volume
+instead of the volume's root. `subPath` allows you to share one volume for
+multiple uses in a single pod.
+
+To view the contents of the volume, run `oc volume pod/<pod_name>`. For example:
```

Comment:
@pmorie Is this what you mean? Showing `oc volume pod`? If so, I can get this example updated to match the Example subPath Usage below. Thanks!